src/IntelliTect.Coalesce/TypeDefinition/ReflectionRepository.cs

```diff
@@ -346,7 +347,9 @@ private void DiscoverNestedCrudStrategiesOn(ClassViewModel model)
         /// <returns></returns>
         public PropertyViewModel PropertyBySelector<T, TProperty>(Expression<Func<T, TProperty>> propertySelector)
         {
-            var objModel = GetClassViewModel<T>();
+            // Nullability note - making a consession here that someone wouldn't do this
+            // on something like .PropertyBySelector<string, int>(s => s.Length).
+            var objModel = GetClassViewModel<T>()!;
```

Comment:
Wouldn't it be better to throw a useful exception in this case rather than a null reference exception on the next line?